configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,33 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    service_hash = [{
+      :directory_ref => settings[:bindir_id],
+      :id => "MCOService",
+      :guid => "7601FCEA-90B3-CC69-6A69-4087FBC7292D",
+      :win64 => settings[:win64],
+      :file => {
+        :id => "RubyWExe",
+        :source => settings[:rubyw],
+      },
+      # This service is installed with start set to demand because
+      # it won't be correctly configured to start right away. The
+      # puppet run that configures mcollective will allow it to start
+      # and set it to automatic
+      :serviceinstall => {
+        :id => "MCOServiceInstaller",
+        :description => "Puppet Labs server orchestration framework",
+        :displayname => "Marionette Collective Server",
```

Comment:
Shouldn't the display name be "Marionette Collective Service" or "Marionette Collective Agent"?.  The MCO Server runs elsewhere.